SwiftUIDZ/SwiftUIDZ/CarsView.swift

```diff
@@ -0,0 +1,246 @@
+//
+//  CarsView.swift
+//  SwiftUIDZ
+//
+//  Created by Анна Сычева on 16.01.2023.
+//
+
+import SwiftUI
+
+/// Экран выбора машин
+struct CarsView: View {
+    
+    // MARK: - Constants
+    
+    private enum Constants {
+        static let meterImageName = "meter"
+        static let seatImageName = "seat"
+        static let engineImageName = "car-engine"
+        static let shareButtonSystemName = "square.and.arrow.up"
+        static let alertRentInfo = "You have successfully rented a car"
+        static let cornerRadius: CGFloat = 8
+        static let commonSize: CGFloat = 150
+        static let rentButtonHeight: CGFloat = 40
+        static let fontName = "Avenir Next"
+        static let fontSizeRegular: CGFloat = 20
+        static let rentButtonTitle = "Rent car"
+        static let totalText = "Total"
+        static let spacing: CGFloat = 30
+        static let smallSpacing: CGFloat = 20
+        static let fontSizeSmall: CGFloat = 15
+        static let fontSizeBig: CGFloat = 100
+        static let kmText = "Km"
+        static let distanceText = "Distance to car"
+        static let rentalText = "Rental Period"
+        static let fillText = "Fill the car?"
+        static let parameterIconSize: CGFloat = 40
+        static let bigImageSize: CGFloat = 300
+        static let oneHundred = 100
+        static let offset = 10
+        static let zero = 0
+        static let zeroDouble: Double = 0
+        static let nameText = "Name:"
+        static let speedText = "Speed:"
+        static let personsText = "Persons:"
+        static let powerText = "Power:"
+    }
+    
+    // MARK: - @State
+    
+    @State private var segmentIndex = 0
+    @State private var segmentIndexRentalPeriod = 0
+    @State private var toggleIsOn = false
+    @State private var progress: Double = 0
+    @State private var isAlertShow = false
+    @State private var isShareShow = false
+    @State private var offset = 0
+    
+    // MARK: - Private properties
+    
+    private var cars = Car.getCars()
```

Comment:
```suggestion
    private var cars = Car.getCars()
    private var rents = Rent.getRens()
```
если они private значит они ниже var body